src/nbl/asset/utils/CSPIRVIntrospector.cpp

```diff
@@ -858,10 +933,14 @@ void CSPIRVIntrospector::CStageIntrospectionData::debugPrint(system::ILogger* lo
         debug << "SPEC CONSTATS:\n";
 
         for (auto& specConstant : m_specConstants)
-            debug << " name: " << std::string_view(specConstant.name.begin(), specConstant.name.end()) << ' '
-            << "TODO: type "
-            << "id: " << specConstant.id << " byte size: " << "TODO: specConstant.defaultValue" << '\n';
-    }
+        {
+            // TODO: it gives weird errors, debug
```

Comment:
maybe doesn't have `\0` at end of span OR it was forgotten about during the mutable->imutable conversion from based_span to regular span

Reply:
fixed?